schemas/2023/08/bundled/outputs/resource/list.json

```diff
@@ -259,7 +260,7 @@
         "input": {
           "$ref": "/PowerShell/DSC/main/schemas/2023/08/definitions/inputKind.json"
         },
-        "implementsPretest": {
+        "preTest": {
```

Comment:
Isn't this renaming reversed?

Reply:
@SteveL-MSFT - I went looking, and it seems like the output schemas were directly updated in #197, instead of the source schemas. When we regenerated them from source, those manual changes were lost. That change happened in this latest release, which I'll address in the next PR - or I can just add all of the schema changes for `alpha.3` into this PR, whichever works better.

Reply:
Went ahead and updated the schemas to reflect the alpha.3 changes.